merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/metamodel/ParameterValidationRecord.java

```diff
@@ -1,13 +1,3 @@
 package gov.nasa.jpl.aerie.merlin.processor.metamodel;
 
-import java.util.Objects;
-
-public final class ParameterValidationRecord {
-  public final String methodName;
-  public final String failureMessage;
-
-  public ParameterValidationRecord(final String methodName, final String failureMessage) {
-    this.methodName = Objects.requireNonNull(methodName);
-    this.failureMessage = Objects.requireNonNull(failureMessage);
-  }
-}
+public record ParameterValidationRecord(String methodName, String[] subjects, String failureMessage) { }
```

Comment:
Git notes that there is a missing newline here. You may want to configure your IDE to ensure all files terminate in a newline.
![image](https://user-images.githubusercontent.com/61392/189010598-dbd0eccf-88dc-4df0-b4de-6a344269ea1b.png)

Reply:
Just this newline left, I think.

Reply:
Oops! Fixed now

Reply:
There's actually one too many newlines now... This is where the "Remove trailing blank lines at the end of saved files" option I also have ticked above helps.

Reply:
Both of those options combined ensure that you'll always have one and no more than one newline at the end of each file.

Reply:
Lol – my bad, I use a mix of command-line editing and IntelliJ and thought I got that one right 😂 Just enabled those settings.

Reply:
Pushed

Reply:
This looks correct now 👀